app/jobs/disability_claim/document_upload.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 class DisabilityClaim::DocumentUpload < ActiveJob::Base
-  # TODO (AJM): Pass in user once it can be serialized using GlobalID
-  def perform(filename, claim_id, tracked_item_id)
-    client = EVSS::DocumentsService.new(User.sample_claimant)
+  def perform(filename, user_attrs, claim_id, tracked_item_id)
+    user = User.new(user_attrs)
+    client = EVSS::DocumentsService.new(user)
```

Comment:
The issue is that AcitveJob can't (de)serialize Time values that are required for the headers (`last_signed_in`), so placing the values in a different object won't address that. The two options I've identified are
- only pass in user attributes that can be serialized by ActiveJob (what I've done here)
- pass in JSON string of user attributes